launchable/utils/session.py

```diff
@@ -0,0 +1,70 @@
+import click
+import os
+import json
+from pathlib import Path
+from typing import Optional
+
+session_file_path = Path('~/.config/launchable/session.json').expanduser()
+
+
+class SessionError(Exception):
+    pass
+
+
+def read_session(build_name: str) -> Optional[str]:
+    if not session_file_path.exists():
+        raise SessionError(
+            "No session file. Please write session {}".format(session_file_path))
+
+    with session_file_path.open('r') as json_file:
+        try:
+            data = json.load(json_file)
+            return data.get("{}:{}".format(build_name, os.getsid(os.getpid())))
+        except json.JSONDecodeError as e:
+            raise SessionError(
+                "Invalid session file. Please remove {}".format(session_file_path))
+
+
+def write_session(build_name: str, session_id: str) -> None:
```

Comment:
If the home directory doesn't exist (which happens typically in a Docker container), or if it's not writable, how does that error get reported? Does that give people enough clue how to solve it? Given how painful sometimes it is to change that, do we provide the escape hatch? Like switching to the explicit session passing with the `--session` option?

Reply:
I added `LAUNCHABLE_SESSION_DIR` environment variable to allow to change the session directory. And notify how to use the variable as an exception message. Do you like this idea?

Reply:
Wow, thank you! I went ahead and tweaked the message a bit.